inc/template-tags.php

```diff
@@ -7,6 +7,31 @@
  * @package _s
  */
 
+if ( ! function_exists( '_s_comment_nav' ) ) :
+/**
+ * Prints comment navigation.
+ */
+function _s_comment_nav() {
+	 // Are there comments to navigate through?
+	if ( get_comment_pages_count() > 1 && get_option( 'page_comments' ) ) : ?>
+		<nav class="navigation comment-navigation" role="navigation">
+			<h2 class="screen-reader-text"><?php esc_html_e( 'Comment navigation', '_s' ); ?></h2>
+			<div class="nav-links">
+				<?php
+					if ( $prev_link = get_previous_comments_link( __( 'Older Comments', '_s' ) ) ) :
+						printf( '<div class="nav-previous">%s</div>', esc_html( $prev_link ) );
```

Comment:
If this would in some form or other be merged at some point, mind that the following will need changing here and below:
`__( 'Older Comments', '_s' )` => `esc_html__( 'Older Comments', '_s' )`
`esc_html( $prev_link )` => `$prev_link`

(as other wise the `<a href=...` would become `&lt;a href=...` etc)